src/app/app.module.ts

```diff
@@ -24,7 +25,8 @@ import { MultiComponent, ErrorMessageComponent } from './shared/ui-components';
     HttpModule,
     FormsModule,
     NgReduxModule,
-    ReactiveFormsModule
+    ReactiveFormsModule,
+    NgbModule.forRoot()
```

Comment:
What is the purpose of this line ?

Reply:
it's important for modal appearance 
https://ng-bootstrap.github.io/#/getting-started